<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 250, 243);</string>
  </property>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 255, 230);</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 255, 230);</string>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 229, 229);</string>
   </property>
   <property name="text">
    <string>☼</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(224, 242, 255);</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
